### PR TITLE
Endpoint to list the XPaths that are applicable to a given VTDNav.

### PR DESCRIPTION
**Pull request by anwar6953**
Endpoint to list the XPaths that are applicable to a given VTDNav.

### CODE REVIEW

File: core/src/main/java/co/cask/wrangler/internal/xpath/XPathUtil.java [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.transform.Transformer;
+import javax.xml.transform.TransformerFactory;
+import javax.xml.transform.dom.DOMSource;
+import javax.xml.transform.stream.StreamResult;
+import javax.xml.transform.stream.StreamSource;
+
+/**
+ *
```
Comment (nitinmotgi) [inline core/src/main/java/co/cask/wrangler/internal/xpath/XPathUtil.java]: Can you add documentation please.

**Comment by nitinmotgi** [comment]
@anwar6953 could you resolve conflicts and merge.

File: core/src/main/java/co/cask/wrangler/internal/xpath/XPathUtil.java (hunk 1 above)
Comment (anwar6953) [reply core/src/main/java/co/cask/wrangler/internal/xpath/XPathUtil.java]: done

**Comment by anwar6953** [comment]
@nitinmotgi resolved conflicts.

**Pull request merged**